VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -970,7 +970,7 @@ Function Restart-ESXiHost {
     # Get the ESXi host uptime before restart.
     $esxiUptime = New-TimeSpan -Start $vmHost.ExtensionData.Summary.Runtime.BootTime.ToLocalTime() -End (Get-Date)
 
-    Restart-VMHost $esxiFqdn
+    Restart-VMHost $esxiFqdn -server $esxiFqdn
```

Comment:
This should be `Restart-VMHost -server $esxiFqdn`

Reply:
No, the command is correct.  the -server denotes which VIserver connection to use.  since there are 2 connections to $esxiFqdn and vCenterServer, the option will defaults to vCenter, however, we disconnected from vCenter so the restart-VMhost will error out.  we need to specify the call is made to $esxiFqdn connection.